scripts/components/versioninfo.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+class VersionInfo {
+	static {
+		$.GetContextPanel().jsClass = this;
+	}
+
+	static onLoad() {
+		this.update();
+	}
+
+	static update() {
+		const cp = $.GetContextPanel();
```

Comment:
as we all know, cp stands for context panel, cyberpunk, and child psychology, nothing else